M/MAGEMin/build_tarballs.jl

```diff
@@ -3,47 +3,35 @@
 using BinaryBuilder, Pkg
 using Base.BinaryPlatforms
 const YGGDRASIL_DIR = "../.."
-include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
+include(joinpath(YGGDRASIL_DIR, "platforms"))
```

Comment:
You're including a directory?  Maybe you wanted to remove this line entirely?

Reply:
Ah yes ok I see. Sorry I am not fully understanding everything with the binaryBuilder.
I removed the line, thank you very much for your help and patience!